dev/SapphireTest.php

```diff
@@ -43,11 +43,6 @@ class SapphireTest extends PHPUnit_Framework_TestCase {
 	 */
 	protected $usesDatabase = null;
 
-	/**
-	 * @deprecated since version 4.0
-	 */
-	protected $originalMailer;
```

Comment:
Have put this back in - do you want me to reinstate the references to it elsewhere? (they literally do nothing now)